scripts/utils/package.js

```diff
@@ -63,27 +63,15 @@ function getPackage(pkg) {
   // Attempt to determine path by pkg name
   let calculatedPackagesDir;
   const fullPathWithPrefix = path.resolve('packages/node_modules', pkg);
-  const sparkFullPath = path.resolve('packages/node_modules/@ciscospark', pkg);
   const webexFullPath = path.resolve('packages/node_modules/@webex', pkg);
   const fullPathWithPrefixExists = isDirectory(fullPathWithPrefix);
-  const sparkPathExists = isDirectory(sparkFullPath);
   const webexPathExists = isDirectory(webexFullPath);
 
   if (fullPathWithPrefixExists) {
     calculatedPackagesDir = fullPathWithPrefix;
   }
-  else {
```

Comment:
We no longer want to auto-resolve the `@ciscospark` packages.

Reply:
Acknowledged, looks good.